docs/guides.rst

```diff
@@ -0,0 +1,293 @@
+Guides and examples
+===================
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. _guide_gateway:
+
+Configuring an Internet Gateway
+-------------------------------
+
+An Internet gateway is a host which is managing the access of a private
+(internal) network to the (external) Internet. When running Linux as a gateway
+host the correct setup of the required iptables rules is crucial. While there
+exist dedicated Linux distributions for this task such as OpenWRT or IPFire,
+it's also possible to use a regular Debian system and configure it through
+DebOps. Here a short overview about the basic steps for a simple gateway
+configuration is given.
+
+The gateway host should have at least two network interfaces connected to the
+respective networks. In this guide the interface named ``eth0`` will be used
+as internal trusted interface and ``eth1`` will be used as external untrusted
```

Comment:
Ok, if you feel that's more consistent, I'll switch that. On my router I give them custom names anyway to make sure I never confuse them. When playing around with Vagrant ``eth0`` was assigned to the management network and ``eth1`` to my custom bridge which I treated as external network, that's why I choose the names like this.